src/content/configuration/resolve.md

```diff
@@ -39,7 +39,7 @@ module.exports = {
 
 ### `resolve.alias`
 
-`object`
+`object` `[string]` `boolean: false`
```

Comment:
`resolve.alias` always takes object I guess.
The changes you are recommending is fit for `resolve.alias[key]`,
cause the changes are recommending

```js
resolve: {
  alias: "string";
  or;
  alias: false;
}
```

Which I think is not a valid.
it should be

```js
resolve: {
  alias: {
    _: string; // or false
  }
}
```

So the description is not describing that. Could you make those changes ?

Reply:
You are completely right, great spot!

Reply:
An array of strings is possible , so it could be any of object and array of strings 

https://github.com/webpack/webpack/blob/21a80704ed4478fbfb07d1b786738c18b4f194bf/schemas/WebpackOptions.json#L2016

Reply:
The all confusion was created because of 

https://github.com/webpack/webpack/blob/21a80704ed4478fbfb07d1b786738c18b4f194bf/schemas/WebpackOptions.json#L2022 

`Resolve.alias.alias`

The second alias accepts `string` `bool` or `[string]`